pennylane/decomposition/decomposition_rule.py

```diff
@@ -602,5 +602,29 @@ def has_decomp(op_type: type[Operator] | str) -> bool:
 
 @register_resources({})
 def null_decomp(*_, **__):
-    """A decomposition rule that does nothing."""
+    """A decomposition rule that does nothing.
```

Comment:
lololol

```suggestion
    """A decomposition rule that instructs the graph-based system to do nothing with the operator(s) it's assigned to.
```

Maybe this is a bit better of an explanation?